armi/utils/mathematics.py

```diff
@@ -57,7 +64,7 @@ def average1DWithinTolerance(vals, tolerance=0.2):
     return avg
 
 
-def convertToSlice(x, increment=False):
+def convertToSlice(x, increment=0):
```

Comment:
With this change, you could consider removing lines 121-122 below:

```
if increment is False:
    increment = 0
```

This would effectively change the API so we might want to avoid that, but I don't see any usage in our codebase where `increment=False` is passed explicitly.

Reply:
Yeah, I was just trying to preserve the old API.

But... I have no idea _why_ anyone would want to pass in `False` here. This is a math function, after all.

Hmmm

Reply:
Okay, I removed that strange `False` from the API, and I am running a big suite of downstream unit tests.

Reply:
Success!  Okay, done.